nbbuild/antsrc/org/netbeans/nbbuild/extlibs/VerifyLibsAndLicenses.java

```diff
@@ -76,6 +79,12 @@ public void setNball(File nball) {
     public void setReport(File report) {
         this.reportFile = report;
     }
+    
+    private File mavenfolderFile;
+    /** JUnit-format XML result file to generate, rather than halting the build. */
+    public void setMavenFolder(File report) {
```

Comment:
This looks strange. The `VerifyLibsAndLicenses` task is pretty specificy. Why is this here? Could this be extracted into a seperate task?

Reply:
I would prefer a separate task - as said generated a pom.xml has little in common with checking licenses (IMHO). They are as near as the already separate download task.